src/garage/tf/algos/ddpg.py

```diff
@@ -283,25 +282,24 @@ def train_once(self, itr, paths):
         if itr % self.steps_per_epoch == 0:
             logger.log('Training finished')
 
-            if self.evaluate:
-                tabular.record('Epoch', epoch)
-                tabular.record('AverageReturn', np.mean(self.episode_rewards))
-                tabular.record('StdReturn', np.std(self.episode_rewards))
-                tabular.record('Policy/AveragePolicyLoss',
-                               np.mean(self.episode_policy_losses))
-                tabular.record('QFunction/AverageQFunctionLoss',
-                               np.mean(self.episode_qf_losses))
-                tabular.record('QFunction/AverageQ', np.mean(self.epoch_qs))
-                tabular.record('QFunction/MaxQ', np.max(self.epoch_qs))
-                tabular.record('QFunction/AverageAbsQ',
-                               np.mean(np.abs(self.epoch_qs)))
-                tabular.record('QFunction/AverageY', np.mean(self.epoch_ys))
-                tabular.record('QFunction/MaxY', np.max(self.epoch_ys))
-                tabular.record('QFunction/AverageAbsY',
-                               np.mean(np.abs(self.epoch_ys)))
-                if self.input_include_goal:
-                    tabular.record('AverageSuccessRate',
-                                   np.mean(self.success_history))
+            tabular.record('Epoch', epoch)
+            tabular.record('AverageReturn', np.mean(self.episode_rewards))
```

Comment:
shouldn't AverageReturn and StdReturn come from `RLAlgorithm.evaluate_performance`?